anchore_engine/services/apiext/swagger/swagger.yaml

```diff
@@ -3465,11 +3465,11 @@ definitions:
     description: A metadata detail record for a specific image. Multiple detail records may map a single catalog image.
     properties:
       created_at:
-        # server returning epoch
-        type: number
+        type: string
+        format: date-time
       last_updated:
-        # server returning epoch
-        type: number
+        type: string
+        format: date-time
```

Comment:
And confirming these are changes to the spec to reflect the formats actually being returned by the service as-is. I confirmed this on a running system.

Reply:
indeed, I tried this out on master with a re-generated client-go and right away the parsing of the response failed due to the these fields. Not a problem since the RFC3339 is the right way 👍